### DISCUSSION TITLE
git completions do not work under zellij

### DISCUSSION THREAD

**Discussion by romange** [zellij-org/zellij#1713] [Q&A]
Hi, for some reason, my git autocompletions stop working with zellij.

I use bash as my shell and my git completions script is `/usr/lib/git-core/git-sh-prompt`. 

Did anyone encounter this problem?

**Comment by riasc** [comment]
Have the same issue... have you found a solution?

**Comment by imsnif** [comment]
Zellij doesn't really deal or change these... most likely culprit IMO is some sort of shell invocation issue (eg. these are set to load by the terminal emulator rather than the shell).
